src/index.ts

```diff
@@ -129,17 +130,37 @@ io.on('connect', (socket: any) => {
     });
   });
 
-  /*socket.on('subscribeChat', (thunderId: string) => {
+  socket.on('subscribeChat', (thunderId: string) => {
     const userId: string = jwt.decode(accessToken) as string;
     const thunder: Promise<ThunderInfo> = chattingHandler.getThunder(userId);
+    thunder.then((thunderInfo: ThunderInfo) => {
+      const tempMember: ObjectId[] = [];
+      thunderInfo.members.forEach(function (chatroomId: ObjectId) {
+        PersonalChatRoom.findOne({_id: chatroomId})
+          .populate('userId')
+          .exec(async (err, foundChatRoom) => {
+            if (err) {
+              throw errorGenerator({
+                msg: message.NOT_FOUND_MEMBER,
+                statusCode: 404,
+              });
+            } else {
+              const foundUserId = foundChatRoom.userId.toString();
+              if (userId === foundUserId) {
+                await chattingHandler.setConnectState(foundChatRoom._id, true);
+                tempMember.push(foundChatRoom._id);
+              } else {
+                tempMember.push(foundChatRoom._id);
+              }
```

Comment:
```suggestion
              if (userId === foundUserId) {
                await chattingHandler.setConnectState(foundChatRoom._id, true);
              }
              tempMember.push(foundChatRoom._id);
```
요렇게 코드 중복을 없앨 수 있을 것 같습니다

Reply:
수정했습니다.